### DISCUSSION TITLE
[Pesquisa de mercado] Como andam as formas de contratação de desenvolvedores?

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#393] [Discussão]
Falaa galera blz?!
Estou fazendo uma pesquisa rápida pra entender um pouco esse mindset, como andam as formas de contratação de desenvolvedores?!
Onde buscam vagas/freelas?
Como são os processos de contratação (testes etc...)
Na visão de vocês, essas plataformas tem um alto indice de assertividade?!

**Comment by rubenmarcus** [comment]
a Printi está usando essa Kenobi http://jobs.kenoby.com/printi
Mas o processo aqui costuma sempre passar pelo RH, depois o gestor da área, e depois o dono faz pessoalmente.


Quanto a vagas acho que ai tem alguns sites: Trampos,GitHub,Linkedin,Stack Overflow Jobs,Jooble,.... aqui no fórum o pessoal falou bastante em algumas issues...

**Comment by guylhermetabosa** [comment]
Aqui é semelhante ao que o @rubenmarcus comentou, passa pelo RH que vai para o diretor responsável que ai sim começa o processo que depois volta pro RH dependendo do feedback do diretor. Linkedin, Infojobs, Facebook sempre tem alguém postando. 

Processo de entrevista + teste em codificação com tempo definido.

**Comment by yanmagale** [comment]
Aqui em BH tem uma startup, [MaxMilhas](https://www.maxmilhas.com.br/busca-passagens-aereas),cujo produto é a venda de passagens através de milhas, sai bem mais barato.

**Comment by leandroruel** [comment]
ja fui em uma empresa que o cara so quis bater um papo e no outro dia ja tava contratado

**Comment by rubenmarcus** [comment]
@leandroruel  Pois é Leandro, e eu ja fui pra vaga especialmente quando é intermediado por rh, que  tinha testes que parecia a fuvest..
E umas 5 entrevistas

recentemente, lembro de uma empresa que o dono me ligava 3x ao dia, falando que ia triplicar meu salário, e que precisava fechar a vaga.. mas pediu um teste pra ser feito.. uma api... fiz.. nunca retornou depois do teste..

acho muito mal caratismo.. Infelizmente é pratica comum do mercado

**Comment by brunoravanhani** [comment]
Eu fiz uma que foi basicamente uma entrevista rápida, depois marcamos outro dia em que me deram um quadro e um pincel atômico e me sabatinaram durante 1h 30m, foi até legal, só que o cara disse que iria ligar na outra quinta-feira e já fazem dois meses.

Mas no geral quando o local da vaga é longe começa um teste prático e depois entrevista via skype/hangouts
